{"name": "tests_delete_account_by_id", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute '_code'", "trace": "accounteapi = <accountapi.accountApi object at 0x027CB220>\n\n    def tests_delete_account_by_id(accounteapi:accountApi):\n        \"\"\" test delete account\n         :param: bookapi->accountApi\n         :returns: MessageModal\n         \"\"\"\n         #The test will fail because of a bug - it says the user is not authorize but it is\n        mylogger.info(\"test for deleting account by id\")\n        dele = accounteapi.delete_account_by_id('98a9f32e-8c82-48ba-ac5b-8c5ca86bc82e')\n>       assert dele._code == '1207'\nE       AttributeError: 'NoneType' object has no attribute '_code'\n\ntestsb.py:86: AttributeError"}, "description": " test delete account\n     :param: bookapi->accountApi\n     :returns: MessageModal\n     ", "start": 1659804795674, "stop": 1659804795916, "uuid": "291285ac-d1b1-4c75-822a-357256c6af0b", "historyId": "52b5e108666504fc314267fb5dae644d", "testCaseId": "b1ce20dc5c0adb254eed92a00a875418", "fullName": "testsb#tests_delete_account_by_id", "labels": [{"name": "suite", "value": "testsb"}, {"name": "host", "value": "VERED-PC"}, {"name": "thread", "value": "3136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testsb"}]}